src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -82,7 +82,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         with self.argument_context(scope, arg_group='Network Rule') as c:
             c.argument('default_action', arg_type=get_enum_type(DefaultAction),
                        help='Default action to apply when no rule matches. Only applicable to Premium SKU.')
-            c.argument('public_network_enabled', get_three_state_flag(), help="Allow public network access for the container registry. The Default is allowed")
+            c.argument('public_network_enabled', get_three_state_flag(), help="Allow public network access for the container registry. The Default is to allow public access.")
+            c.argument('allow_trusted_services', get_three_state_flag(), is_preview=True, help="Allow trusted Azure Services to access network restricted registries. For more information, please visit https://aka.ms/acr/trusted-services. The default is to allow.")
```

Comment:
Help message says `The default is to allow`.

For `acr update`, there should be no default values for the options in update commands.
For `acr create`, I don't see the logic to default to allow. Is it handled on service side?

Reply:
yes, this is the default on the service side. Should I remove this?

Reply:
You can keep `The default is to allow` for `acr create`, but should remove it for `acr update`. If `allow_trusted_services` is none in update command, then the setting will not be changed.

Reply:
yes this makes sense. I see what you mean. Thanks for the clarification